usdoj-crt/beta-ada
# Moving print.js into webpack build pipeline.

Moving print.js script from main.js //require statement (Jekyll Assets/Sprockets) to the Webpack build process. 

This is step one in moving all of our legacy JS into the modern Webpack build pipeline.

To test:
1. Open the preview link below
2. Attempt to print a page with accordions on it. DO not open the accordions beforehand.
3. The accordions should be expanded in the print preview
4. View the page source and confirm that the print-compiled.js script appears to be compressed.

Addresses step 1 in the procedure outlined in this ticket: https://github.com/usdoj-crt/beta.ADA.gov-Project-Management/issues/501
